client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,223 @@
  * External dependencies
  */
 import { bindActionCreators } from 'redux';
-import { localize } from 'i18n-calypso';
+import { translate } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
+import { fetchEmailSettings } from 'woocommerce/state/sites/settings/email/actions';
 import {
-	fetchEmailSettings,
-} from 'woocommerce/state/sites/settings/email/actions';
+	getEmailSettings,
+	areEmailSettingsLoading,
+	areEmailSettingsLoaded,
+} from 'woocommerce/state/sites/settings/email/selectors';
+import Card from 'components/card';
+import CustomerNotification from './components/customer-notification';
+import InternalNotification from './components/internal-notification';
+import NotificationsOrigin from './components/notifications-origin';
+
+const fromName =
+	{
+		field: 'woocommerce_email_from_address',
+		title: translate( 'From name' ),
+		subtitle: translate( 'Emails will appear in recipients inpboxses \'from\' this name.' ),
+	};
+const fromAddress =
+	{
+		field: 'woocommerce_email_from_name',
+		title: translate( 'From address' ),
+		subtitle: translate( 'If recipients reply to store emails they will be sent to this address.' ),
+	};
+
+const internalNotifications = [
+	{
+		field: 'email_new_order',
+		title: translate( 'New order' ),
+		subtitle: translate( 'Sent when a new order is received.' ),
+	},
+	{
+		field: 'email_cancelled_order',
+		title: translate( 'Cancelled order' ),
+		subtitle: translate( 'Sent when a new order is marked \'cancelled\'.' ),
+	},
+	{
+		field: 'email_failed_order',
+		title: translate( 'Failed order' ),
+		subtitle: translate( 'Sent when a new order is marked \'failed\'.' ),
+	},
+];
+
+// id enabled
+const customerNotifications = [
+	{
+		field: 'email_customer_on_hold_order',
+		title: translate( 'Order pending payment' ),
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'Refunded order' ),
```

Comment:
This should be `New account` with a subtitle of `Sent when customers sign up via checkout or account page`